src/core/tests/frontend/paddle/test_models/gen_scripts/generate_assign.py

```diff
@@ -0,0 +1,32 @@
+import os
```

Comment:
I think we need to add copyright message

Reply:
Done.